articles/how-to-use-local-storage-with-JavaScript/index.md

```diff
@@ -0,0 +1,284 @@
+//in this case, the `grade` is the key while `One` is the value.
+```
+
+As mentioned before, we must `stringify` objects before we store them in the local storage. An example is outlined below.
+
+```JavaScript
+const Car = {
+  brand:"Suzuki",
+  color:"white",
+  price:10000
+}
+
+Window.localStorage.setItem('car', JSON.stringify(Car));
+```
+
+Failure to stringify the object will result in an error.
+
+#### getItem()
+This function is used to access or retrieve the data in the local storage. The method takes in a `key` as a parameter. It then extracts the required value from the local storage.
+
+For example, to retrieve the above `Car` object, we will use the following statement.
+
+```JavaScript
+Window.localStorage.getItem('car');
+```
+
+The above statement will return something like this;
+
+```JavaScript
+"{brand:"Suzuki",color:"white",price:"10000"}"
+```
+
+You should convert it to an object using `JSON.parse()` to use it in your code.
+
+```JavaScript
+JSON.parse(window.localStorage.getItem('car'));
+```
+
+#### removeItem()
+This method is used to delete an item from the local storage. The `removeItem()` method requires a key as a parameter.
+
+```javascript
+window.localStorage.removeItem('brand');
+```
+
+#### clear()
+This method is used to clear all values stored in the local storage. It does not require any parameters.
+
+```JavaScript
+Window.localStorage.clear()
+```
+
+### Project
+Now that we have learned about the primary functions of local storage, let's create a web application that stores, retrieves, deletes, and clears items from the local storage.
+
+Create a new folder and open it in your code editor. Create two files, `index.html` and `main.js`. The `index.html` will showcase the web page to the user, while the `main.js` will store our javascript functions. These functions will be used to access different functionalities of the `Local Storage`.
+
+### Let's Code
+Our `index.html` will have a `form` and several `buttons`, as shown below.
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>localStorage</title>
+  <link rel="stylesheet" href="./css/bootstrap.css">
+  <script type="text/javascript" src="main.js"></script>
+
+</head>
+
+<body>
+  <header>
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
+        aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarCollapse">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <br>
+  <br>
```

Comment:
This navbar isn't related to local storage. 

Remove this. We must keep the code very simple for the readers to understand easily.